zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterEventClient.java

```diff
@@ -211,6 +318,42 @@ public void putResponseGetResource(String resourceId, ByteBuffer object) {
 
   /**
    * Supposed to call from RemoteInterpreterEventPoller
+   * @param invokeMessage json serialized InvokeMessage
+   * @param object java serialized of the object
+   */
+  public void putResponseInvokeMethod(
+      InvokeResourceMethodEventMessage invokeMessage, ByteBuffer object) {
+    Object o = null;
+    try {
+      o = Resource.deserializeObject(object);
+    } catch (IOException e) {
+      logger.error(e.getMessage(), e);
+    } catch (ClassNotFoundException e) {
+      logger.error(e.getMessage(), e);
```

Comment:
Still need to put null into `getInvokeResponse`, otherwise, https://github.com/apache/zeppelin/pull/1951/files#diff-5d91426e574ad85664eb7a7857ef6e90R207 may not finish.